ref/general/authentication.adoc

```diff
@@ -34,18 +38,30 @@ Open Liberty supports the following options for SSO authentication:
 
 - link:/docs/ref/feature/#openidConnectServer-1.0.html[OpenID Connect]  +
-OIDC is a popular internet SSO protocol that works well with cloud-native applications. This protocol enables a client application to request the identity of the user as an ID token in a standardized, REST-like manner. The client application, or Relying Party, requests information about a user from an authorization server, known as an OpenID Connect Provider. After users are authenticated by an OpenID Provider, they can access any Relying Party that is configured for that Provider, without having to manage unique accounts for each application or website.
+OIDC is a popular internet SSO protocol that works well with cloud-native applications. This protocol enables a client application to request the identity of the user as an ID token in a standardized, REST-like manner. The client application, or Relying Party, requests information about a user from an authorization server, which is known as an OpenID Connect Provider. After users are authenticated by an OpenID Provider, they can access any Relying Party that is configured for that Provider, without having to manage unique accounts for each application or website.
 
 - link:/docs/ref/general/#/docs/concept/sso-config-saml.html[Security Assertion Markup Language (SAML 2.0)] +
 SAML Web Browser SSO enables web applications to delegate user authentication to a SAML identity provider instead of a configured user registry.
 
 - link:/docs/ref/feature/#spnego-1.0.html[Simple and Protected GSS-API Negotiation Mechanism (SPNEGO)] +
 With SPNEGO SSO, users log in to the Microsoft Domain controller once to access and move between protected resources on Open Liberty servers without getting prompted for additional logins.
 
+= JAAS authentication with Open Liberty
+
+Open Liberty relies on the Java Authentication and Authorization Service (JAAS) framework to determine a user's identity and access roles. JAAS authentication is suited to microservice-based applications because it separates the authentication process from the rest of the application so that authentication can be managed independently. New or updated authentication processes can be plugged in to an application without otherwise modifying the application itself.
+
+The JAAS framework defines a `Subject` object to represent the source of a request to an application. The subject can be any entity that makes a request, such as a user or another microservice. The subject contains a set of principals, or details about the identity of the entity that made the request, which is used for authentication. The subject also contains sets of credentials, or security attributes that are used for authorization. By creating a subject to represent a user or service, Open Liberty provides an efficient and easily configurable means for authentication data to pass between microservices in an application.
```

Comment:
I think this isn't right:

>The subject contains a set of principals, or details about the identity of the entity that made the request, which is used for authentication.
specifically I don't think the or should be there since you are describing what a principal is, or implies an alternative rather than a definition.

I think credentials are optional in a subject.